### PR TITLE
Add the attosoc test from nextpnr-xilinx

### PR DESCRIPTION
**Pull request by HackerFoo**

### CODE REVIEW

File: project/attosoc.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    ],
+    "top": "top",
+    "name": "attosoc",
+    "toolchains": {
+        "vivado": {
+            "arty": ["arty.xdc"]
+        },
+        "nextpnr": {
+            "arty": ["arty.xdc"]
+        }
```
Comment (litghost) [inline project/attosoc.json:17]: Should we have an issue to try adding vpr here once IO LOC support for symbiflow-arch-defs is added (e.g. https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1519 )

Comment (acomodi) [reply project/attosoc.json:14]: I believe we can temporarily use PCF and convert it to XDC once the support is ready.

On a side note, with https://github.com/SymbiFlow/fpga-tool-perf/pull/136, nextpnr toolchain moved to be `nextpnr-xilinx` as I have added ice40 support (which resulted in the need renaming nextpnr toolchains).

File: project/picorv32.json [hunk 2]
```diff
@@ -13,6 +13,9 @@
         "vpr": {
             "arty": ["arty.pcf"],
             "zybo": ["zybo.pcf"]
+        },
+        "nextpnr": {
```
Comment (acomodi) [inline project/picorv32.json:17]: `nextpnr-xilinx`

**Comment by acomodi** [comment]
I believe the failure in CI is related to an outdated version of Yosys. As soon as the master+wip branch gets updated, this should turn green.

**Comment by mithro** [comment]
@acomodi - We should resurrect or close out this pull request.

**Pull request closed without merging**